source/whats-new.txt

```diff
@@ -293,10 +293,9 @@ What's New in 4.8
 
 .. important:: Deprecation Notice
 
-   The ``IterableCodec`` and ``MapCodec`` classes are deprecated. Instead,
-   use ``CollectionCodec`` and ``MapCodecV2``.
-   These support any class that implements ``Collection`` or
-   ``Map``.
+   The ``IterableCodec`` and ``MapCodec`` classes are deprecated.
+   Use ``MapCodecProvider`` instead of ``MapCodec`` and ``CollectionCodecProvider``
+   instead of ``IterableCodec``.
```

Comment:
Suggestion for clarity:

```suggestion
   The ``IterableCodec`` and ``MapCodec`` classes are deprecated.
   Use ``MapCodecProvider`` instead of ``MapCodec``, and ``CollectionCodecProvider``
   instead of ``IterableCodec``.
```